tasks/run_cd4pe_job.rb

```diff
@@ -102,7 +102,7 @@ def initialize(web_ui_endpoint:, job_token:, ca_cert_file: nil, logger:)
 
   def make_request(type, api_url, payload = '')
     connection = Net::HTTP.new(@http_config[:server], @http_config[:port])
-    if @http_config[:scheme] == 'https'
+    if @http_config[:scheme] == 'https' and !@ca_cert_file.nil?
```

Comment:
does it make sense for a unit test?

Reply:
totally, but yeah it may not happen just this minute

Reply:
Next time we should write the failing until test first :) ...